.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
Please remove this file. 

You can use [global gitignore](https://help.github.com/en/github/using-git/ignoring-files) for cases like these.

Reply:
yeah I'm sorry, that was unintentional